packages/dso-toolkit/src/components/annotation/annotation.args.ts

```diff
@@ -1,65 +1,211 @@
+    control: {
+      type: "text",
+    },
+  },
+  locatieNoemers: {
+    control: {
+      type: "array",
+    },
   },
-): Annotation<TemplateFnReturnType> {
+};
+
+export function annotationActiviteitArgsMapper(a: AnnotationActiviteitArgs): Annotation {
   return {
-    annotationButton: {
-      identifier: a.identifier,
-      open: a.open,
-      dsoClick: (e) => a.dsoClick(e.detail),
+    type: "activiteit",
+    ...a,
+  };
+}
+
+/**
+ * Gebiedsaanwijzing
+ */
+
+export interface AnnotationGebiedsaanwijzingArgs extends AnnotationArgsBase {
+  naam: AnnotationDiff | string;
+}
+
+export const annotationGebiedsaanwijzingArgs: Omit<AnnotationGebiedsaanwijzingArgs, "dsoActiveChange"> = {
+  ...annotationBaseArgs,
+  naam: "Winkelgebied",
+};
+
+export const annotationGebiedsaanwijzingArgTypes: ArgTypes<AnnotationGebiedsaanwijzingArgs> = {
+  ...annotationArgTypesBase,
+  naam: {
+    control: {
+      type: "text",
     },
-    annotationOutput: {
-      identifier: a.identifier,
-      open: a.open,
-      prefix: a.prefix,
-      title: annotationContent.title,
-      addons: annotationContent.addons,
-      content: annotationContent.content,
-      dsoClose: (e) => a.dsoClose(e.detail),
+  },
+};
+
+export function annotationGebiedsaanwijzingArgsMapper(a: AnnotationGebiedsaanwijzingArgs): Annotation {
+  return {
+    type: "gebiedsaanwijzing",
+    ...a,
+  };
+}
+
+/**
+ * Omgevingsnorm
+ */
+
+export interface AnnotationOmgevingsnormArgs extends AnnotationArgsBase {
+  naam: AnnotationDiff | string;
+  waardes: Array<AnnotationDiff | string>;
+  eenheid: AnnotationDiff | string;
+}
+
+export const annotationOmgevingsnormArgs: Omit<AnnotationOmgevingsnormArgs, "dsoActiveChange"> = {
+  ...annotationBaseArgs,
+  naam: "Geluid",
+  waardes: [{ verwijderd: "50", toegevoegd: "45" }, "55"],
+  eenheid: "dB",
+};
+
+export const annotationOmgevingsnormArgTypes: ArgTypes<AnnotationOmgevingsnormArgs> = {
+  ...annotationArgTypesBase,
+  naam: {
+    control: {
+      type: "text",
     },
+  },
+  waardes: {
+    control: {
+      type: "array",
+    },
+  },
+  eenheid: {
+    control: {
+      type: "text",
+    },
+  },
+};
+
+export function annotationOmgevingsnormArgsMapper(a: AnnotationOmgevingsnormArgs): Annotation {
+  return {
+    type: "omgevingsnorm",
+    ...a,
+  };
+}
+
+/**
+ * Werkingsgebied
+ */
+
+export interface AnnotationWerkingsgebiedArgs extends AnnotationArgsBase {
+  locatieNoemers: Array<AnnotationDiff | string>;
+}
+
+export const annotationWerkingsgebiedArgs: Omit<AnnotationWerkingsgebiedArgs, "dsoActiveChange"> = {
+  ...annotationBaseArgs,
+  locatieNoemers: [
+    { verwijderd: "bedrijven categorie A" },
+    "winkelgebied",
+    { toegevoegd: "woonwijk" },
+    { was: "poldergebied", wordt: "tuingebied" },
+  ],
+};
+
+export const annotationWerkingsgebiedArgTypes: ArgTypes<AnnotationWerkingsgebiedArgs> = {
+  ...annotationArgTypesBase,
+  locatieNoemers: {
+    control: {
+      type: "select",
```

Comment:
Denk dat dit `array` moet zijn.

<img width="1702" alt="Screenshot 2024-05-31 at 09 27 06" src="https://github.com/dso-toolkit/dso-toolkit/assets/4646174/d3d2b43f-3509-42f0-b2b4-b150b0af1ac0">

Reply:
Goed gevonden. Ik ga dit aanpassen, maar het zal anders worden dan nu: Een werkingsgebied kan volgens mij maar één locatieNoemer hebben.

Reply:
Ik heb er één locatieNoemer van gemaakt. `locatieNoemers` is dus `locatieNoemer` geworden. De control type is nu `text`.